squirrel/template.py

```diff
@@ -0,0 +1,44 @@
+"""This module contains the class to store stellar and other templates and process
+them."""
+
+import numpy as np
+from .data import Spectra
+
+
+class Template(Spectra):
```

Comment:
Is it a good idea to have a single `Template` class that handles both the more "usual" stellar template libraries and other perhaps less usual quasar template or other custom templates? Is the initialization of these two categories of templates the size the same?
Maybe it is, but I just wanted to make sure it is adapted.

Reply:
For now, I think, both types of templates can be stored in this same class unless there are fundamentally different properties that the different templates need to store. The `flux` variable of this class can have both 1D or 2D arrays. However, we can make child classes if it becomes clearer that having different class names would be more user-friendly or convenient.